### DISCUSSION TITLE
Openthread update plan consultation

### DISCUSSION THREAD

**Discussion by aaa147147** [openthread/openthread#10972] [Q&A]
What is the update plan for OpenThread? Our code repository is from July 1, 2023 and is used for product development

I noticed that there have been a total of 1044 updates since July 1, 2023, but I haven't checked the details because there are too many updates

![image](https://github.com/user-attachments/assets/f54bb720-01ce-4a34-b074-03c1e3ada271)

May I ask if this includes any important updates that I must update, as well as the time for the next major version update

**Comment by jwhui** [comment]
I suggest updating to the latest main branch to pull in all the latest bug fixes and enhancements, including Thread 1.4.

**Comment by aaa147147** [comment]
Thank you for your reply and suggestions. We will update to the latest version.
